gatsby-config.ts

```diff
@@ -164,6 +166,51 @@ const gatsbyConfig: GatsbyConfig = {
                     <img src="${imageUrl}" alt="">
                   `;
 
+                // combine intro text and copy/body text
```

Comment:
There is now a lot of code in the file. Can you try to extract it and put it somewhere under `/gatsby`?

Reply:
I did some refactoring in 8c7d6729bbeb6f8850c899647498becb108eac0c to remedy this.